frontend/src/components/header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import './Header.css'
+import {HeaderProps} from "./props";
+
+const Header: React.FC<HeaderProps> = ({isLogged}: HeaderProps) => {
+    return (
+        <header className="header">
+            <nav>
+                <ul>
+                    {isLogged && <li><NavLink to="/profile">Profile</NavLink></li>}
+                    {isLogged && <li><NavLink to="/topUp">TopUp</NavLink></li>}
+                    {isLogged && <li><NavLink to="/transactions">Transactions</NavLink></li>}
+                    {!isLogged && <li><NavLink to="/signin">Sign in</NavLink></li>}
```

Comment:
```typescript

{isLogged &&
                        <div>
                            <li><NavLink to="/profile">Profile</NavLink></li>
                            <li><NavLink to="/topUp">TopUp</NavLink></li>
                            <li><NavLink to="/transactions">Transactions</NavLink></li>
                        </div>
                    }
 ``` 
Or you can show everything all the time, as your routes don't allow to move forward without logging